setup.py

```diff
@@ -8,6 +8,7 @@
 extras_require = {
     'tensorflow': ['tensorflow<2'],
     'pytorch': ['torch'],
+    'webknossos': ['webknossos'],
```

Comment:
The release cycles of webknossos are quite short with regular breaking changes. It is probably better to hard-code working release.